src/scss/_arrow.scss

```diff
@@ -34,6 +49,10 @@
 	}
 }
 
+/// Top arrow styling
+///
+/// @param {Color} color
```

Comment:
The default color `#74736c` is `pink-tint5`, should it be requiring o-colors instead of having it hard coded?